ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/CVEsTable.tsx

```diff
@@ -165,6 +169,14 @@ function CVEsTable({
                     >
                         Top CVSS
                     </TooltipTh>
+                    {isNvdCvssEnabled && (
+                        <TooltipTh
+                            // sort={getSortParams('TODO', aggregateByTODO)}
+                            tooltip="Highest CVSS score (from National Vulnerability Database) of this CVE across images"
```

Comment:
What do you think about this:

```The highest CVSS score (from the National Vulnerability Database) for this CVE across all images.```

Reply:
Although I think it is more fluent, adapted directly from `toolTip` text for **CVSS** column (which was not apparently, because adjacent column was in unchanged lines).